aten/src/ATen/xpu/EmptyTensor.cpp

```diff
@@ -0,0 +1,96 @@
+#define TORCH_ASSERT_NO_OPERATORS
+#include <ATen/Context.h>
+#include <ATen/EmptyTensor.h>
+#include <ATen/xpu/EmptyTensor.h>
+#include <c10/core/DeviceGuard.h>
+
+namespace at::detail {
+
+TensorBase empty_xpu(
+    IntArrayRef size,
+    ScalarType dtype,
+    c10::optional<Device> device_opt,
+    c10::optional<c10::MemoryFormat> memory_format_opt) {
+  const auto device = device_or_default(device_opt);
+  TORCH_INTERNAL_ASSERT(device.is_xpu());
+  // XXX
+  // const c10::DeviceGuard device_guard(device);
+  auto* allocator = at::getCPUAllocator();
```

Comment:
Hack... Are you going to fix it in this PR?

Reply:
Yes. It depends on runtime PRs landed.